starter/script.js

```diff
@@ -1 +1,125 @@
 // put code here!
+var name = prompt("What\'s your name?");
+console.log(name);
+
+var gender = prompt("Hi, " + name + ". You have the day off today!  This script is going to help decide what to do.  First, we have to ask: are you male or female? (M/F)");
+//console.log(gender);
+ //**(M/F)**
+
+if (gender === 'F' || gender === 'f'){
```

Comment:
how can we limit this to just `(gender == "f")`? if we lowercase the users input, we only have to have one portion of the conditional.